tensorflow/python/ops/image_ops_impl.py

```diff
@@ -211,6 +211,45 @@ def _EnsureTensorIs4D(image):
 
   return (image, is_batch)
 
+def _flip_image(image, axis, random=False, seed=None):
+  """
+  Flips image(s) around a given axis.
+
+  Args:
+    image: 4-D Tensor of shape `[batch, height, width, channels]` or
+           3-D Tensor of shape `[height, width, channels]`.
+
```

Comment:
I think you need to get rid of this blank line for the comment to render properly